test/lib/ngzip/builder_test.rb

```diff
@@ -5,23 +5,27 @@
 describe Ngzip do
 
   it 'must support the static method :build' do
-    Ngzip.respond_to?(:build).must_equal true
+    expect(Ngzip.respond_to?(:build)).must_equal true
```

Comment:
Isn't this kind of double testing? I think `(foo).must_equal true` is enough for testing the boolean result here which is returned by `respond_to?`

Reply:
Oh, this change is to address a warning that MiniTest generates: https://stackoverflow.com/a/58128950/2517033

Reply:
Ah, I see - did only know the "_()" version before. As I use rspec matchers also with minitest this seemed like a duplication.

Reply:
@dup2 LMK if anything else on this you want changed!